### PR TITLE
Fixed #438 - Moved toFixed() function inside the generateBytes functi…

### LINKED ISSUE #438: FileManager: Cannot read property 'toFixed' of null
**Client or Server side error** 
Server error
Error octofarm

**Describe the bug**
The bug occurred when I added a Prussa MK3 printer to OctoFarm. There is alot of files on the printer which showed up in OctoPrint. 

**To Reproduce**
Do not know how to reproduse 

**Expected behavior**
To see all the files on the printer

**Screenshots**
Error octofarm

**Versions**

Docker: 19.03.13
{
  "Id": "sha256:ffc588835399000ca916346b7398f8b4745d6b288185f29d28679a5a034c5462",
  "Digest": null,
  "RepoDigests": [
    "octofarm/octofarm@sha256:70f366079ab038078a51912729f4e133735b5d09f707a24b4587b74f1cd650b0"
  ],
  "Labels": {
    "org.opencontainers.image.created": "2021-03-05T07:50:06.839Z",
    "org.opencontainers.image.description": "OctoFarm is an  web server and client combination for unifying multiple instances of Octoprint. You can manage and monitor as many instances as you want from a single interface giving you full control over your 3D printer farm.",
    "org.opencontainers.image.licenses": "AGPL-3.0",
    "org.opencontainers.image.revision": "d5983dc8ca175c580380fb3585531bea6e280360",
    "org.opencontainers.image.source": "https://github.com/OctoFarm/OctoFarm",
    "org.opencontainers.image.title": "OctoFarm",
    "org.opencontainers.image.url": "https://github.com/OctoFarm/OctoFarm",
    "org.opencontainers.image.version": "master"
  }
}

 - OS: Zorin 15.3
 - (docker container) OctoFarm Version 1.1.12
 - (docker container) OctoPrint Version 1.5.3
 - Firmware: Prusa-Firmware 3.9.3 based on Marlin

**Additional context**
I suspect that there is something in the file names that might be filtered out, or alternatively a off by one error.

### PR DESCRIPTION
**Pull request by NotExpectedYet**
…on in pretty helpers. Stops the page failing when trying toFixed a string.

### CODE REVIEW

File: views/partials/functions/pretty.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const bytes = function (a, b) {
       d = b || 2,
       e = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"],
       f = Math.floor(Math.log(a) / Math.log(c));
-    return parseFloat((a / Math.pow(c, f)).toFixed(d)) + " " + e[f];
+    return parseFloat((a / Math.pow(c, f)).toFixed(d)).toFixed(2) + " " + e[f];
```
Comment (davidzwa) [inline views/partials/functions/pretty.js:13]: - first tofixed **string** on division result
- then to float numeric
- back to tofixed string

please clean up ;)

**Review by davidzwa — CHANGES_REQUESTED** [review]
template is up to you. I would rename the generateBytes function name to something which indicates that it results in:
- a string ouput
- byte size order

generateNumBytesString() ok with u?

**Comment by NotExpectedYet** [comment]
> template is up to you. I would rename the generateBytes function name to something which indicates that it results in:
> 
>     * a string output
> 
>     * byte size order
> 
> 
> generateNumBytesString() ok with u?

I don't see the point currently XD

**Review by davidzwa — APPROVED** [review]
LGTM didnt test as I dont have a lot of data on my printers

**Comment by NotExpectedYet** [comment]
> LGTM didnt test as I dont have a lot of data on my printers

The new client will hopefully rest your OCD with function names.

**Pull request merged**